### PR TITLE
Projects/ Favs Style Updates

### LINKED ISSUE #1078: Update 'projects/ favs' section styling in profile view
Need to update the 'projects/ favs' section underneath the profile overview page.

Current styling:
Screen Shot 2019-05-27 at 3 09 41 PM

Updates:
Screen Shot 2019-05-27 at 3 08 56 PM
Screen Shot 2019-05-27 at 3 09 04 PM

cc @youriwims @mmmavis @alanmoo

### PR DESCRIPTION
**Pull request by youriwims**
Closes #1078 

This is a shared component that is used 👇 :
1. [Projects/Favs tabs](https://network-pulse-staging-pr-1084.herokuapp.com/profile/556) _If you can't access this link, just login_
2. [Not Found pages](https://network-pulse-staging-pr-1084.herokuapp.com/profile/)
3. [Empty Bookmark page](https://network-pulse-staging-pr-1084.herokuapp.com/favs)
4. [Add Page](https://network-pulse-staging-pr-1084.herokuapp.com/add) _Need to be logged out to view_
5. [Entry Submission page](https://network-pulse-staging-pr-1084.herokuapp.com/submitted)

### CODE REVIEW

**Comment by mmmavis** [comment]
@youriwims is it safe if I press `update branch`?

**Comment by youriwims** [comment]
Go for it 🙃

File: components/hint-message/hint-message.jsx [hunk 1]
```diff
@@ -11,9 +11,8 @@ class HintMessage extends React.Component {
 
   render() {
     return (
-      <div className="hint-message text-center">
-        <h2 className="icon mb-3">{ this.props.iconComponent }</h2>
-        <h2>{ this.props.header }</h2>
+      <div className="pt-4 pt-lg-5 text-center">
```
Comment (mmmavis) [inline components/hint-message/hint-message.jsx]: Not a huge issue here as it's just spacing but we should get into a habit of mentioning all the elements affected to design reviewers if we changed a shared component code. This way we don't accidentally introduce design changes we didn't want.

---

`pt-4` (1.5rem top padding) and `pt-lg-5` (3rem top padding for larger screen) don't correspond to the old spacing we set in `hint-message.scss` (3rem / 5rem). As this `hint-message.jsx` component is also used outside of the profile/favs tabs we should get Nat to design review all the `<HintMessage>` occurrences on the site to make sure the new spacing look right on other pages as well.

Comment (youriwims) [reply components/hint-message/hint-message.jsx]: good point, i'll update the initial comment and tag Nat so she can review the other instances.

**Comment by youriwims** [comment]
@natalieworth, @mmmavis brought up a good point [here](https://github.com/mozilla/network-pulse/pull/1084#discussion_r288780835). I've updated the initial comment to reflect where else this component (the hint message) is used. I can already see that I would need to go in and update some of them to match the projects/favs update. But, I just want to confirm with you first. 

Should all of these instances share the same style? Once you let me know, I'll go in and update and re-request your review to make sure we've covered anything. 

Screen Shot 2019-05-30 at 1 37 58 PM

**Comment by youriwims** [comment]
@natalieworth ⬆️

**Pull request merged**